run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Public Functions
+
+// RunContainer runs a container with given ID
+func RunContainer(containerID string, containerPath string) error {
```

Comment:
Then call then Systemd containers.  The point I am after it we need to support the equivalent of
docker run -ti fedora sh
Which would probably execute RunContainer()  :^)